models/repo.go

```diff
@@ -651,6 +651,12 @@ func (repo *Repository) GetMirror() (err error) {
 	return err
 }
 
+// UpdateMirror updates the repository mirror, returns an error upon failure
+func (repo *Repository) UpdateMirror(m *Mirror) (err error) {
```

Comment:
what is this func for?

we already have UpdateMirror

Reply:
This is now removed.